### PR TITLE
Switch MDC Catalog to a black and white theme

### PR DESCRIPTION
**Pull request by liquidx**
This change makes it in line with the visual style for Material. 

simulator screen shot may 2 2017 3 19 30 pm
simulator screen shot may 2 2017 3 19 34 pm
simulator screen shot may 2 2017 3 19 44 pm

### CODE REVIEW

File: catalog/MDCCatalog/MDCCatalogComponentsController.swift [hunk 1]
```diff
@@ -87,7 +87,7 @@ class MDCCatalogComponentsController: UICollectionViewController, MDCInkTouchCon
 
     let titleLabel = UILabel()
     titleLabel.text = self.title!.uppercased()
-    titleLabel.textColor = UIColor(white: 0.46, alpha: 1)
```
Comment (willlarche) [inline catalog/MDCCatalog/MDCCatalogComponentsController.swift]: Needs f for float

File: catalog/MDCCatalog/MDCCatalogTiles.m [hunk 2]
```diff
@@ -27,13 +27,21 @@
   return image;
 }
 
+static UIColor *MDCPrimaryColor() {
+  return [UIColor colorWithWhite:0.2 alpha:1];
```
Comment (willlarche) [inline catalog/MDCCatalog/MDCCatalogTiles.m]: Needs f for float

File: catalog/MDCCatalog/MDCCatalogTiles.m [hunk 3]
```diff
@@ -27,13 +27,21 @@
...
+}
+
+static UIColor *MDCSecondaryColor() {
+  return [UIColor colorWithWhite:0.7 alpha:1];
```
Comment (willlarche) [inline catalog/MDCCatalog/MDCCatalogTiles.m]: Needs f for float

**Review by willlarche — CHANGES_REQUESTED** [review]
Looks great!

1. I requested 'f' be next to 2 numbers to allow the tests to pass (CGFloat warning.)
2. Is there no use for the highlight green?

File: catalog/MDCCatalog/MDCCatalogTiles.m (hunk 3 above)
Comment (liquidx) [reply catalog/MDCCatalog/MDCCatalogTiles.m]: Done

File: catalog/MDCCatalog/MDCCatalogTiles.m (hunk 2 above)
Comment (liquidx) [reply catalog/MDCCatalog/MDCCatalogTiles.m]: Done

**Comment by liquidx** [comment]
I didn't add the green yet, since the current screens were already monochrome, but was in blue instead of black.

I have some followups with the actual examples that will use the accent color.

**Review by willlarche — APPROVED** [review]
Tests passed!

**Pull request merged**